app/src/main/java/com/example/itbook/data/source/BooksDataSource.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.itbook.data.source
+
+import com.example.itbook.data.model.Book
+import com.example.itbook.utils.OnDataLoadCallBack
+
+interface BooksDataSource {
+    interface remote {
+        fun getBooksFromAPI(query: String, callback: OnDataLoadCallBack<List<Book>>)
```

Comment:
Nằm trong Remote rồi thì không cần fromAPi nữa
```suggestion
        fun getBooks(query: String, callback: OnDataLoadCallBack<List<Book>>)
```